tests/data/snapshots/tests/SnapshotDisplayDiffCest.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+use Snapshot\UserSnapshot;
+
+class SnapshotDisplayDiffCest
+{
+    public function snapshotCanDisplayDiff(DataTester $I, UserSnapshot $snapshot)
+    {
+        $snapshot->assert();
+
+        $expected = $I->grabColumnFromDatabase('users', 'email');
+        $actual = array_merge($expected, ['hobgoblin@vasya.com']);
+
+        $I->haveInDatabase('users', [
+           'name' => 'hobgoblin',
+           'email' => 'hobgoblin@vasya.com'
+        ]);
+
+        $snapshot->shouldRefreshSnapshot(false);
+        try {
+            $snapshot->shouldShowDiffOnFail();
+            $snapshot->assert();
```

Comment:
I think that you need `$I->fail()` after assert for completeness, to cover the case when assert doesnt throw exception.

Reply:
I've pushed it while github was having issues, so it its not showing up in the PR yet, idk why.

https://github.com/fkupper/Codeception/commit/0b7fa057e7eca0af19107c0cf2a7164bb34d7402

Reply:
Force pushed after amending, it works.